### ISSUE TITLE
What's New: Update Ticket Reference URLs

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#5476]
Since issues are now hosted at GitHub, all URLs referenced in the What's New file should be updated to point to the new issue URLs. This should be as easy as simply replacing all occurrences of "community.nvda-project.org/ticket" with "https://github.com/nvaccess/nvda/issues".
